src/Core/Entities/UserSignatureKeyPair.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
❓ Should more of these go under code ownership?

Reply:
Moved this to KM ownership